packages/dd-trace/test/dogstatsd.spec.js

```diff
@@ -179,4 +223,65 @@ describe('dogstatsd', () => {
     expect(udp6.send.firstCall.args[3]).to.equal(7777)
     expect(udp6.send.firstCall.args[4]).to.equal('::1')
   })
+
+  const udsIt = os.platform() === 'win32' ? it.skip : it
+  udsIt('should support HTTP via unix domain socket', (done) => {
```

Comment:
Let's wrap the test in a condition instead. This makes it easier to manually skip/only when needed. Skipping tests is also usually meant for a test that will be implemented in the future or is temporarily disabled, not a test that will never run which just pollutes the output.

Reply:
That has not been my experience. Other test frameworks have a `todo(...)` function for that and `skip(...)` is used specifically for conditional inclusion of tests like platform support.

Reply:
Other test frameworks might use these words for other things, but from the Mocha documentation: "Anything skipped will be marked as [pending](https://mochajs.org/#pending-tests), and reported as such."

Reply:
Pending tests are ones without a test function, not ones marked skip. Those are two different concepts.

Reply:
Already `.only` still has a test running N times due to having N versions tested. You could argue that it's the same test, and therefore semantically correct, but I really think this is mostly just hair splitting.

To be honest: Pending isn't the worst thing here, since nothing should really stop us from supporting named pipes on Windows, but we haven't tested it anywhere else and the main reason I was skipping it is I ran into permission issues on GH Actions, and I decided it wasn't worth figuring out right now.

Reply:
This was really just a nitpick and I didn't expect it to become a larger conversation. Let's keep it as it is.